### PR TITLE
[#9382] Add tests for status-message component

### PR DESCRIPTION
**Pull request by Pengyuz**
Part of #9382 

Add tests for status-message component.

### CODE REVIEW

File: src/web/app/components/status-message/status-message.component.spec.ts [hunk 1]
```diff
@@ -19,7 +20,50 @@ describe('StatusMessageComponent', () => {
...
+    fixture.detectChanges();
+    const statusMessages: HTMLElement[] = fixture.nativeElement.querySelectorAll('div.alert div');
+    expect(statusMessages.length).toEqual(3);
+    expect(statusMessages[0].innerHTML).toEqual('a');
+    expect(statusMessages[1].innerHTML).toEqual('b');
+    expect(statusMessages[2].innerHTML).toEqual('c');
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should delete a message after clicking button', () => {
```
Comment (xpdavid) [inline src/web/app/components/status-message/status-message.component.spec.ts]: Try to separate/grouping the `setup`, `execution` and `verification` parts by inserting emtpy lines in the test?

File: src/web/app/components/status-message/status-message.component.spec.ts [hunk 2]
```diff
@@ -19,7 +20,49 @@ describe('StatusMessageComponent', () => {
...
+    const buttons: HTMLElement[] = fixture.nativeElement.querySelectorAll('div.alert button');
+
+    buttons[0].click();
+    fixture.detectChanges();
+
+    const statusMessagesAfterClick: HTMLElement[] = fixture.nativeElement.querySelectorAll('div.alert div');
+
+    expect(statusMessagesAfterClick.length).toEqual(2);
+    expect(statusMessagesAfterClick[0].innerHTML).toEqual('b');
+    expect(statusMessagesAfterClick[1].innerHTML).toEqual('c');
```
Comment (xpdavid) [inline src/web/app/components/status-message/status-message.component.spec.ts]: Can also verify `component.messages`?

File: src/web/app/components/status-message/status-message.component.spec.ts [hunk 3]
```diff
@@ -19,7 +20,49 @@ describe('StatusMessageComponent', () => {
...
+
+    buttons[1].click();
+    buttons[2].click();
+    fixture.detectChanges();
+
+    const deleteAll: HTMLElement[] = fixture.nativeElement.querySelectorAll('div.alert div');
+
+    expect(deleteAll.length).toEqual(0);
```
Comment (xpdavid) [inline src/web/app/components/status-message/status-message.component.spec.ts]: Same here?

**Comment by Pengyuz** [comment]
Suggested changes are applied. Ready for review.

**Pull request merged**